lms/models/_mixins.py

```diff
@@ -0,0 +1,15 @@
+import sqlalchemy as sa
+
+
+class TimestampedModelMixin:
```

Comment:
Super nit pick, but this isn't necessarily a model / the fact it's a model comes from elsewhere. So I think this is `TimestampedMixin` or `WithTimestamps` or something.

Reply:
:+1:  for something like `CreatedUpdatedMixin`

Reply:
@jon-betts We could do `CreatedMixin` and `UpdatedMixin` but I don't we need those independently as we might for Accessed, what do you think?

Reply:
I think we should follow the usage. If created and updated always go together, then the class should too. I can't think of a good reason why we wouldn't want these both if we ever have either. If we do find that happens, it's easy to split up.

The `accessed` does seem like a different kettle of fish, as we have to code specifically to support it wherever it's used, so it makes sense to me for that not to be a part of it.